Add sentence about degenerate case (#15942)
mdn/content: files/en-us/web/api/clearinterval/index.md

The global **`clearInterval()`** method cancels a timed, repeating action which
was previously established by a call to {{domxref("setInterval", "setInterval()")}}.

> If the identifier passed in parameter has not been previous set,

Reviewer: ```suggestion
If the parameter provided does not identify a previously established action,
```
